static/updatenotification.sh

```diff
@@ -25,6 +25,16 @@ NCMIN=$(curl -s -m 900 $NCREPO/ | sed --silent 's/.*href="nextcloud-\([^"]\+\).z
 REPORTEDMAJ="$CURRENTVERSION"
 REPORTEDMIN="$CURRENTVERSION"
 
+# Check for supported Nextcloud version
+if [ "${CURRENTVERSION%%.*}" -lt "$NCBAD" ]
```

Comment:
Hope that works now

Reply:
Looks better yes. :+1: 

But is this really needed? https://github.com/nextcloud/vm/blob/287c83682425654c40e92c0649aa8cc4be1a164a/static/updatenotification.sh#L24-L26

Reply:
> But is this really needed?
> 
> https://github.com/nextcloud/vm/blob/287c83682425654c40e92c0649aa8cc4be1a164a/static/updatenotification.sh#L24-L26

You mean if the linked code is needed?

Reply:
Exactly.

Reply:
It is indeed needed for getting the next minor update version so that we are able to send a notification if either a new minor or a major version is found. See the logic below.

Reply:
Reportedmaj and reportedmin store tje already reported version. Which is need so thqt you don't get an updatenotificarion each day but only one time for each version...